build/ci/github-actions/share-image-jobs.md

```diff
@@ -47,9 +47,6 @@ jobs:
     runs-on: ubuntu-latest
     needs: build
     steps:
-      -
-        name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v2
```

Comment:
I don't have a lot of hands-on experience with GHA, but how is the `docker load` command available after this change? @crazy-max

Reply:
Good catch @YubinMoon

@dvdksn Docker is available on public runners, no need to setup buildx to use the cli.